feat: Ad hoc filters support multi-value operators

Loori-R opened the pull request:
* Added `"multiValueFilterOperators": true` flag in `plugin.json` to enable `one of` (`=|`) and `not one of` (`!=|`) operators in Ad hoc filters UI - available in Grafana starting from **v11.3.x**
* Implemented query translation (expressions are passed via the `extra_filters` parameter):
  * `one of` (`=|`) → `key:in("value1", "value2", ..., "valueN")`
  * `not one of` (`!=|`) → `!(key:in("value1", "value2", ..., "valueN"))`

Related #293

> [!IMPORTANT]  
> When using a **datasource variable**, multi-value operators will **not be shown** in Ad hoc filters due to a Grafana issue (see [grafana/grafana#110465](https://github.com/grafana/grafana/issues/110465)).

<img width="507" height="363" alt="image" src="https://github.com/user-attachments/assets/f36b902c-ad0a-48a9-9f2f-cf908fd31c3c" />

arturminchukov commented:
Cannot test it on grafana@10.4.19. Should it works on such version of grafana?
On v12.x.x - everything is fine.

Loori-R commented:
Updated the PR:  
- Fixed the `_stream` filter handling.  
- Clarified that multi-value operators are available in Grafana starting from **v11.3.x** (see [grafana/grafana#91837](https://github.com/grafana/grafana/pull/91837)).  
- Updated the changelog and description.

arturminchukov reviewed (approved):
Thanks! LGTM!
